### PR TITLE
Removed Animations

### LINKED ISSUE #1548: Remove animation from view buttons
**Actual Behaviour**

The icons for code view, conversation view and tree view on Build tab of botbuilder has an animation for helper text on hover. Same is for Code view and UI view on Design tab and Configure tab.

**Expected Behaviour**

These animations need to be removed.

**Screenshots of the issue**

screen shot 2018-08-14 at 7 19 03 pm

**Would you like to work on the issue?**

This is a small issue good for beginners. So if anyone wants to take this up, just comment here and then send a PR.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1548

Surge Deployment Link: https://pr-1754-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:

### CODE REVIEW

**Comment by sk9331657** [comment]
@DNS-404 @Akshat-Jain @akshatnitd  Please review.

**Comment by sk9331657** [comment]
@Akshat-Jain  @akshatnitd Please review.

**Comment by sk9331657** [comment]
@DNS-404  I need to override the styling else animations won't be removed. Can you please check how can I resolve the failing Codacy.??

**Comment by sk9331657** [comment]
@fossasia/susi-ai-skills-cms-devs  Can you please review this ??

**Comment by sk9331657** [comment]
@DNS-404  Thanks. I have the pr. Please check it.

File: src/components/BotBuilder/BotBuilderPages/Animation.min.css [hunk 1]
```diff
@@ -0,0 +1 @@
+.iconbutton,.iconbutton>div,.iconbutton>div>div,.iconbutton>svg{transition:none!important}
```
Comment (DNS-404) [inline src/components/BotBuilder/BotBuilderPages/Animation.min.css]: Add a newline.

File: src/components/SkillCreator/Animation.min.css [hunk 2]
```diff
@@ -0,0 +1 @@
+.iconbutton,.iconbutton>div,.iconbutton>div>div,.iconbutton>svg{transition:none!important}
```
Comment (DNS-404) [inline src/components/SkillCreator/Animation.min.css]: Add a newline.

Comment (sk9331657) [reply src/components/SkillCreator/Animation.min.css]: Done

File: src/components/BotBuilder/BotBuilderPages/Animation.min.css (hunk 1 above)
Comment (sk9331657) [reply src/components/BotBuilder/BotBuilderPages/Animation.min.css]: Done

**Comment by sk9331657** [comment]
@DNS-404  Please review now.

**Comment by sk9331657** [comment]
@DNS-404  Also review my pending pr's.

**Comment by sk9331657** [comment]
@akshatnitd  Please review .

**Comment by sk9331657** [comment]
@akshatnitd This can be merged now.

**Comment by sk9331657** [comment]
@DNS-404  This can also be merged.

**Pull request merged**